REST/PowerShell/Variables/FindVariableUsage.ps1

```diff
@@ -141,10 +139,32 @@ foreach ($project in $projects)
     }
 }
 
+$VariableSets = (Invoke-RestMethod -Method Get "$OctopusURL/api/libraryvariablesets?contentType=Variables" -Headers $header).items
```

Comment:
Some customers may have a large number of variable sets, so I think this could be beneficial behind a switch e.g

```ps
if ($searchVariableSets -eq $True) { 

# ... Your new code searching variable sets
}
```

The default behavior should be to have this set to `$False` e.g at the top of the script after https://github.com/OctopusDeploy/OctopusDeploy-Api/pull/106/files#R19

```ps
# Search through Variable Set values?
$searchVariableSets = $True
```